resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/KubernetesVolumeUtils.scala

```diff
@@ -16,9 +16,12 @@
  */
 package org.apache.spark.deploy.k8s
 
+import scala.util.Try
```

Comment:
If you use the given helper function, we don't need to use `Try`.

Reply:
sure!